### ISSUE TITLE
anchors with delete method do not redirect

### ISSUE THREAD

**Issue by scottbarrow** [ParamagicDev/mrujs#73]
#### Expected Behaviour
Clicking on 
`<a class="text-danger" data-method="delete" data-confirm="Are you sure you want to delete this contact?" data-turbo="false" href="/sellers/2/contacts/10">Delete</a>`
is expected to make a `delete` request to the server, where the controller action performs a redirect, and the page respond with a redirect

Controller
```ruby
  def destroy
    @contact.destroy

    flash[:success] = 'Contact Deleted'
    redirect_to seller_contacts_path(current_organization)
  end
```

#### Actual Behaviour
The controller action is hit and the browser receives the response to redirect, however the DOM remains unchanged

#### Steps to Reproduce
1. Create an anchor with a delete method, confirm text and data-turbo="false"
2. Create a controller destryo action that responds with a redirect

**Comment by ParamagicDev** [comment]
Well that's not good! Let me work on reproducing this locally.

**Comment by ParamagicDev** [comment]
Huh... @scottbarrow I tried reproducing this locally in the mrujs dummy rails app and cant reproduce this behavior. Do you have a repo I could checkout? Or a minimal recreation of the problem?

**Comment by ParamagicDev** [comment]
@leastbad seems to be encountering the same issue. Gonna dig into this and see whats up.

**Comment by ParamagicDev** [comment]
@scottbarrow pretty sure I found the issue. Pushing up a fix shortly.

**Comment by ParamagicDev** [comment]
@scottbarrow pretty sure this was due to bad querySelectors on my part.

This should be fixed now in `0.3.0.beta-29`

Feel free to reopen this if it doesnt fix it.
